src/bin/ipc_latency.rs

```diff
@@ -88,18 +134,45 @@ macro_rules! netlink_bench {
                 .output()
                 .expect("make failed to start");
 
-            let (tx, rx) = mpsc::channel::<Vec<Duration>>();
+            let (tx, rx) = mpsc::channel::<Vec<(Duration, Duration, Duration)>>();
 
             // listen
             let c1 = thread::spawn(move || {
                 let mut buf = [0u8; 1024];
-                let nl = portus::ipc::netlink::Socket::<$mode>::new()
-                    .map(|sk| {
-                        Backend::new(sk, Arc::new(atomic::AtomicBool::new(true)), &mut buf[..])
-                    })
+                // What does the backend represent?
```

Comment:
`Backend` represents a wrapper around `Ipc` impls that deals in messages instead of bytes (i.e. a serialization wrapper layer).

On the receiving side, it also deals with message batching.

Reply:
But this comment should probably be removed